core/trino-main/src/main/java/io/trino/cost/CboFeaturesConfig.java

```diff
@@ -41,6 +42,7 @@
     private boolean collectPlanStatisticsForAllQueries;
     private boolean ignoreStatsCalculatorFailures = true;
     private boolean defaultFilterFactorEnabled;
+    private double filterConjunctionIndependenceFactor = 0.75;
```

Comment:
Have you tried other constants?

Reply:
The existing code is similar to setting this to 1
I had tried 0.5 as well
I both cases there were regressions to some TPC queries with default filter factor enabled